app/src/androidTest/java/org/mozilla/fenix/ui/SmokeTest.kt

```diff
@@ -274,9 +274,10 @@ class SmokeTest {
 
         navigationToolbar {
         }.enterURLAndEnterToBrowser(defaultWebPage.url) {
+            mDevice.waitForIdle()

```

Comment:
Not sure if we are using this in other tests or there is a "nicer" way to add that wait within the tests.

Reply:
Actually, this approach is used in quite often in our tests.
That's the main reason I've used it.